scripts/build-debianpackage

```diff
@@ -51,10 +51,6 @@ mkdir -p "${TOP_BUILDDIR}/${PKG_NAME}"
 
 printf "Building package '%s' from version '%s'...\\n" "$PKG_NAME" "$PKG_VERSION"
 
-# Update changelog interactively. Currently using "unstable" but we intend
-# to target "stretch" for SDW applications. Must run before files are copied
-# into non-version-controlled build dir.
-dch --newversion "${PKG_VERSION}" --distribution unstable -c "${CUR_DIR}/${PKG_NAME}/debian/changelog"
```

Comment:
The reason we added this auto-dch command is two-fold:

1. It's extremely easy to overlook doing it yourself, thus the script; but we can learn to run a command ad-hoc, it won't kill us.
2. There are a bunch of flags that should be added to the `dch` invocation, particularly the `--distribution`, that's quite painful to recall from memory on every run.

I'll read the manpage for dch completely before suggesting any changes; maybe tossing in a `scripts/dch-wrapper` would accommodate the manual workflow with flags...